companies/tests.py

```diff
@@ -1,68 +1,80 @@
-from django.test import TestCase
 from rest_framework.test import APITestCase, APIClient
-from django.urls import reverse
-from authentication.models import CustomUser
+from rest_framework import reverse
+from authentication.models import CustomUser, CompanyAndUserRelation, Company
+from forum.managers import TokenManager
+from forum.settings import FRONTEND_URL
 
 
 class CompanyTestAuthenticatedUser(APITestCase):
 
     def setUp(self):
-        self.company_url = reverse('companies-list-create')
-        user = CustomUser.objects.create_user('test@mail.com','Test_password123456')
+        self.company_url = f'{FRONTEND_URL}/companies/'
+        self.get_company_url =  f'{self.company_url}get_company/'
```

Comment:
U can use reverse method for simplify url work, what do you think?

Reply:
unfortunatly, here DRF's viewset and router are used, so reverse() doesn't realy works